<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMySQL</class>
 <widget class="QMainWindow" name="QtMySQL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtMySQL</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1,1,0">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" colspan="2">
        <widget class="QTableView" name="tableView"/>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Filter</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioMale">
            <property name="text">
             <string>male</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radioFemale">
            <property name="text">
             <string>female</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioAll">
            <property name="text">
             <string>all</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>arrangement</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1" colspan="2">
           <widget class="QComboBox" name="comboBoxArrange"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QRadioButton" name="radioAscend">
            <property name="text">
             <string>ascend</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QRadioButton" name="radioDescend">
            <property name="text">
             <string>descend</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>arrange</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Sex</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="2" colspan="2">
        <widget class="QSpinBox" name="spinBoxAge"/>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QComboBox" name="comboBoxSex"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>ID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Birth</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Age</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="2" colspan="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxSalary"/>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Country</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QSpinBox" name="spinBoxID"/>
       </item>
       <item row="8" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEditCountry"/>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Salary</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="2">
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="LabImage">
         <property name="text">
          <string>picture</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionAdd"/>
   <addaction name="actionInsert"/>
   <addaction name="actionDelete"/>
   <addaction name="actionSet_Picture"/>
   <addaction name="actionClear_picture"/>
   <addaction name="actionBonus"/>
   <addaction name="actionSave"/>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/open.bmp</normaloff>:/icons/open.bmp</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/316.bmp</normaloff>:/icons/316.bmp</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/306.bmp</normaloff>:/icons/306.bmp</iconset>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/delete.bmp</normaloff>:/icons/delete.bmp</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSet_Picture">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/31.ico</normaloff>:/icons/31.ico</iconset>
   </property>
   <property name="text">
    <string>Set Picture</string>
   </property>
  </action>
  <action name="actionClear_picture">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/103.bmp</normaloff>:/icons/103.bmp</iconset>
   </property>
   <property name="text">
    <string>Clear picture</string>
   </property>
  </action>
  <action name="actionBonus">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/724.bmp</normaloff>:/icons/724.bmp</iconset>
   </property>
   <property name="text">
    <string>Bonus</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/save.bmp</normaloff>:/icons/save.bmp</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/undo.bmp</normaloff>:/icons/undo.bmp</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/exit.bmp</normaloff>:/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>QtMySQL</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
